### PR TITLE
Randomization Params

### PR DESCRIPTION
**Pull request by Saurav-D**
Added SynthDet Randomization Param doc for public UnityGroceries - Synthetic dataset.

### CODE REVIEW

File: docs/UnityGroceriesSyntheticRandomizationParams.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# UnityGroceries - Synthetic Randomization Parameters
+
+SynthDet follows a curriculum strategy for generating data. The full dataset is 
+generated using a sequence of scale factors, generally starting with objects 
+that are larger and gradually growing smaller. For a complete description of the 
+SynthDet curriculum, see the following paper: https://arxiv.org/pdf/1902.09967.pdf. 
+For public UnityGroceries - Synthetic dataset we generated nearly around 5k
```
Comment (JonathanHUnity) [inline docs/UnityGroceriesSyntheticRandomizationParams.md:7]:
```suggestion
For the UnityGroceries - Synthetic dataset we generated almost 5k
```
Should clarify if this means "almost" or "about" 5k images

**Review by JonathanHUnity — CHANGES_REQUESTED** [review]
This page should be linked from the rest of the documentation. Since the description of this specific dataset is in datasetinsights now, should we split this doc into two - one which describes the parameters and one which describes UnityGroceries - Synthetic?

**Comment by JonathanHUnity** [comment]
@Saurav-D This is a really nice addition. Can we get this landed?

**Comment by Saurav-D** [comment]
> @Saurav-D This is a really nice addition. Can we get this landed?

Sure. Sorry, I got off track.

**Comment by mkamalza** [comment]
This PR was made for the original SynthDet project, which is now living in the SynthDet_Original branch. I am going to close this for now as the new master branch has a vastly different project.

**Pull request closed without merging**